src/components/TodoList/index.tsx

```
@@ -199,89 +173,145 @@ export const TodoList = memo(
+                  {doneTodoList.map((doneTodo, idx) => (
+                    <MemoTodoCard
+                      isThisEdit={editTodoId === doneTodo.id}
+                      setEditTodoId={setEditTodoId}
+                      todoData={doneTodo}
+                      focusStep={focusStep}
+                      randomTagColor={randomTagColor}
+                      isCurrTodo={false}
+                      order={idx + 1}
+                    />
+                  ))}
+                </List>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)